src/state/disconnect-toast-mixin.ts

```diff
@@ -9,6 +9,10 @@ import { HassBaseEl } from "./hass-base-mixin";
 
 export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
   class extends superClass {
+    private _subscribed?: Promise<() => Promise<void>>;
+
+    private _dismissed = false;
```

Comment:
As this is a mixin, we should make these var names more specific

Reply:
adjusted 👍